YoutubeExplode/Videos/VideoController.cs

```diff
@@ -42,14 +42,10 @@ public VideoController(HttpClient http)
         );
     }
 
-    public async Task<PlayerResponseExtractor> GetPlayerResponseAsync(
+    public async Task<PlayerResponseExtractor> GetPlayerResponseAndroidClientAsync(
```

Comment:
Can you use the previous signature, i.e. `GetPlayerResponseAsync(VideoId videoId, bool isEmbedded, CancellationToken cancellationToken = default)` and just get the `TVEMBED` version if `isEmbedded` is `true`? To reduce code churn a bit.

Reply:
sure

Reply:
When using the TV embed client we need to send the signatureTimestamp from the player in the request, Do I add another parameter for the signatureTimestamp?